user_guides/jakartaee/src/main/jbake/content/intro.adoc

```diff
@@ -459,38 +504,39 @@ Linux software that meet the following software requirements:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In addition to the instructions and requirements described in this
-document, all Jakarta EE 10 and Jakarta EE 10 Web Profile implementations must
+document, all Jakarta EE 10 Platform implementations must
 also pass the standalone TCKs for the following technologies:
 
-* Jakarta Batch 2.1
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Dependency Injection 2.0
-* Jakarta Bean Validation 3.0
-* Jakarta XML Binding 3.0
-* Jakarta JSON Binding 3.0
-* Jakarta JSON Processing 2.1
-* Jakarta RESTful Web Services 3.1
-
-For more information about the Jakarta Batch technology,
-see the specification at `https://jakarta.ee/specifications/batch/2.1/`
-
-For more information about the Jakarta Contexts and Dependency Injection technology, 
-see the specification at `https://jakarta.ee/specifications/cdi/4.0/`
-
-For more information about the Jakarta Dependency Injection,
-see the specification at `https://jakarta.ee/specifications/dependency-injection/2.0/` 
-
-For more information about the Jakarta Bean Validation technology, see the
-specification at `https://jakarta.ee/specifications/bean-validation/3.0/`
-
-For more information about the Jakarta XML Binding technology, see the
-specification at `https://jakarta.ee/specifications/xml-binding/3.0/`
-
-For more information about the Jakarta JSON Binding technology, see the specification at `https://jakarta.ee/specifications/jsonb/3.0/`
-
-For more information about the Jakarta JSON Processing technology, see the specification at `https://jakarta.ee/specifications/jsonp/2.1/`
+* Jakarta Activation -- see https://jakarta.ee/specifications/activation/ for additional details
+* Jakarta Authentication -- see https://jakarta.ee/specifications/authentication/3.0/ for additional details
+* Jakarta Batch -- see https://jakarta.ee/specifications/batch/2.1/ for additional details
+* Jakarta Bean Validation -- see https://jakarta.ee/specifications/bean-validation/3.0/ for additional details
+* Jakarta Concurrency -- see https://jakarta.ee/specifications/concurrency/3.0/ for additional details
+* Jakarta Contexts and Dependency Injection -- see https://jakarta.ee/specifications/cdi/4.0/ for additional details
+* Jakarta Debugging Support for Other Languages -- see https://jakarta.ee/specifications/debugging/ for additional details
```

Comment:
For EE 9.1, I see Debugging Support in list of technologies tested by Platform TCK (e.g. from `1.2.1 Jakarta EE 9.1 Technologies Tested with Jakarta EE 9.1 Platform TCK` list) but not as a required Standalone TCK.